civihr_employee_portal/views/includes/civihr_employee_portal_handler_aggregated_personal_address.inc

```diff
@@ -12,6 +12,7 @@ class civihr_employee_portal_handler_aggregated_personal_address extends views_h
    */
   public function render($values) {
     $contactID = isset($values->id) ? $values->id : NULL;
+    $contactID = !$contactID && isset($values->civicrm_address_contact_id) ? $values->civicrm_address_contact_id : $contactID;
```

Comment:
Why is contact ID attained differently now?

Reply:
@mickadoo Honestly I can't answer that question right now with the current time constrains. I checked exactly for that difference too, but in dashboard is retrieved as property `id` and in My Details page is retrieved as another property.